gnome-initial-setup/pages/account/um-photo-dialog.c

```diff
@@ -176,16 +179,33 @@ get_facesdirs (void)
         const gchar * const * data_dirs;
         int i;
         const char *facesdir_env;
+        g_autoptr(GKeyFile) keyfile = NULL;
+        g_autoptr(GError) error = NULL;
 
         facesdir_env = g_getenv ("GIS_ACCOUNT_PAGE_FACESDIR");
         if (facesdir_env != NULL && g_strcmp0 (facesdir_env, "") != 0) {
                 facesdirs = g_slist_prepend (facesdirs,
                                              g_strdup (facesdir_env));
         }
 
-        if (g_strcmp0 (ACCOUNT_PAGE_FACESDIR, "") != 0) {
-                facesdirs = g_slist_prepend (facesdirs,
-                                             g_strdup (ACCOUNT_PAGE_FACESDIR));
+        keyfile = g_key_file_new ();
+        if (!g_key_file_load_from_file (keyfile, VENDOR_CONF_FILE, G_KEY_FILE_NONE, &error)) {
```

Comment:
Don't reload the keyfile here. We have API for this!

```c
g_autofree gchar *path = gis_driver_conf_get_string (driver, VENDOR_ACCOUNT_GROUP, VENDOR_ACCOUNT_FACESDIR_KEY);
if (path != NULL)
  facesdir = g_slist_prepend (facedir, path);
```

Reply:
Please also fix cc-common-language.c to use this. I see that a3910b9395410dd8d35620f235ff2c957610e6e8 made it re-parse the file.